config/default.yml

```diff
@@ -17,6 +17,26 @@ AllCops:
   # as the default.
   TargetRailsVersion: ~
 
+Lint/NumberConversion:
+  # Add Rails' duration methods to the ignore list for `Lint/NumberConversion`
+  # so that calling `to_i` on one of these does not register an offense.
```

Comment:
It may be good to add the address below :-)

```suggestion
  # Add Rails' duration methods to the ignore list for `Lint/NumberConversion`
  # so that calling `to_i` on one of these does not register an offense.
  # See: https://github.com/rubocop-hq/rubocop/issues/8950
```